File: frontend/orderly/src/main/java/com/yebur/model/request/OrderDetailRequest.java
package com.yebur.model.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class OrderDetailRequest {
    private Long productId;
    private Long orderId;
    private String name;
    private String comment;
    private int amount;
    private BigDecimal unitPrice;
    private String status;
    private String paymentMethod;
    private String batchId;
    private LocalDateTime createdAt;
    private Long cashSessionId;
}

File: frontend/OrderlyPhone/app/src/main/java/com/example/orderlyphone/domain/model/request/OrderDetailRequest.kt
package com.example.orderlyphone.domain.model.request

import java.math.BigDecimal
import java.time.LocalDateTime

data class OrderDetailRequest(
    val productId: Long,
    val orderId: Long,
    val name: String,
    val comment: String?,
    val amount: Int,
    val unitPrice: BigDecimal,
    val status: String,
    val paymentMethod: String?,
    val batchId: String,
    val createdAt: LocalDateTime,
    val cashSessionId: Long
)
